wger/email/migrations/0002_auto_20180503_1220.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2018-05-03 09:20
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('email', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='log',
+            options={'ordering': ['-date']},
+        ),
+        migrations.AlterField(
+            model_name='log',
+            name='gym',
+            field=models.ForeignKey(editable=False, on_delete=django.db.models.deletion.CASCADE, related_name='email_log', to='gym.Gym'),
```

Comment:
This line looks long. Can I suggest installing flake8 in your text editor? We will be using flake8 style guide throughout the project. I feel it would be good to have it on your machine too. What do you think?

Reply:
this is autogenerated migration file

Reply:
Ooh. I see.